src/CMakeLists.txt

```diff
@@ -521,15 +521,21 @@ if(${TRITON_ENABLE_TRACING})
   if (NOT WIN32)
     target_include_directories(
       tracing-library
-      PRIVATE ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+      PUBLIC ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
```

Comment:
@fpetrini15 any general wisdom on necessity of PUBLIC vs PRIVATE and why it fixes it? or @KrishnanPrash

Reply:
IIUC, by specifying `tracing-library`'s dependencies as private, any other targets that depend on `tracing-library` will not be able to see its dependencies (not inherited)--they will have to link them again individually. In this case, it appears that to compile the `tracing-library`, CMake needs to have access to the open telemetry include directories. From my meetings with @KrishnanPrash, it seems like anything that depends on `tracing-library` also needs to have access to this directory. In this case, it would make sense to expose this as a public dependency.

Take this with a grain of salt, however, as I am no CMake expert 😅

Reply:
Changing from PRIVATE to PUBLIC allows the directories to be included with any library that links to the tracing-library, which in this case is py-bindings.